addons/web_editor/tests/test_tools.py

```diff
@@ -87,20 +87,22 @@ def test_get_video_url_data(self):
         for key in ['vimeo', 'vimeo_player']:
```

Comment:
Commit message:
- First commit:
  - the commit title should start with a verb such that it completes the sentence "When applied, this commit will ..." so here "enhanced" is not right :wink: 
  - You left the two task references, I think only one is needed now that the commits are separated :slightly_smiling_face: 
 
- Second commit:
  - same comment for the task references :+1: